static/js/groups_page.js

```diff
@@ -72,13 +72,16 @@ var groupsPage = {
         submit: function (e) {
             e.preventDefault();
 
+             $('#CreateGroupBtn').attr('disabled', true);
+
             var group = {
                 "name": $("#GroupNameInput").val() == "" ? undefined : $("#GroupNameInput").val(),
                 "description": $("#GroupDescriptionInput").val() == "" ? undefined : $("#GroupDescriptionInput").val(),
                 "owner_email": $('#UserEmail').val()
             };
 
             if (!group['name']) {
+                $('#CreateGroupBtn').attr('disabled', false);
                 return alert("Please enter in a valid group name!");
```

Comment:
I just realized we are still using alert here. Can you use `$.notify` to alert errors?

Reply:
@adbharadwaj There are many alerts in this code. Should I change them all or just the one relevant to this issue?

Reply:
Just relevant to this issue.